x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_list_table.tsx

```diff
@@ -145,10 +156,90 @@ export const AgentListTable: React.FC<Props> = (props: Props) => {
         );
       },
     },
+    ...(displayAgentMetrics
+      ? [
+          {
+            field: 'metrics',
+            sortable: false,
+            name: (
+              <EuiToolTip
+                content={
+                  <FormattedMessage
+                    id="xpack.fleet.agentList.cpuTooltip"
+                    defaultMessage="Average CPU usage in the last 5 minutes"
+                  />
+                }
+              >
+                <span>
+                  <FormattedMessage id="xpack.fleet.agentList.cpuTitle" defaultMessage="CPU" />
+                  &nbsp;
+                  <EuiIcon type="iInCircle" />
+                </span>
+              </EuiToolTip>
+            ),
+            width: '75px',
+            render: (metrics: AgentMetrics | undefined, agent: Agent) =>
+              metrics?.cpu_avg && metrics?.cpu_avg !== 0 ? (
+                `${(metrics.cpu_avg * 100).toFixed(2)} %`
+              ) : (
```

Comment:
Hmm since this is repeated verbatim I wonder if a small utility would be helpful here. Would be annoying to accidentally break this math in one place and not the other, for example.